README.md

```diff
@@ -3,29 +3,34 @@
 </a-->
 
 # Codewind for VS Code
+Create and develop cloud-native, containerized web applications from VS Code.
 
 [![Marketplace](https://img.shields.io/vscode-marketplace/v/IBM.codewind.svg?label=marketplace&logo=visual-studio-code)](https://marketplace.visualstudio.com/items?itemName=IBM.codewind)
 [![License](https://img.shields.io/badge/License-EPL%202.0-red.svg?label=license&logo=eclipse)](https://www.eclipse.org/legal/epl-2.0/)
 [![Slack](https://img.shields.io/badge/ibm--cloud--tech-blue.svg?logo=slack&label=slack)](https://slack-invite-ibm-cloud-tech.mybluemix.net/)
 
-- **[Documentation](https://microclimate.dev/codewindtechpreview)**
-- **[Changelog](https://github.com/microclimate-dev2ops/codewind-vscode/blob/master/CHANGELOG.md)**
-
-Create and develop cloud-native, containerized web applications from VS Code.
-
-## Getting started
+## Installing Codewind
+Prerequisites
+- Install [VS Code Version 1.27 or later](https://code.visualstudio.com/download).
 
-1. Install [VS Code version 1.27 or later](https://code.visualstudio.com/download).
-2. Install Codewind for VS Code from the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or by searching for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+<Add gif file here. The gif will begin with a title card and then cover all of the actions in the following section.>
+Complete both parts of the Codewind installation.
+1. Find Codewind for VS Code in the [VS Code Marketplace](https://marketplace.visualstudio.com/items?itemName=IBM.codewind-tools) or search for "Codewind" in the [VS Code Extensions view](https://code.visualstudio.com/docs/editor/extension-gallery#_browse-for-extensions).
+2. From the Codewind page in the VS Code Extensions Marketplace, click the **Install** button. The first part of Codewind installs.
+3. Go to the **Explorer** view group and open the **Codewind** view. The Codewind workspace is now added to the Explorer menu.
+4. To complete the second part of the installation, expand the Codewind workspace and click any of the functions. A notification window appears and asks if you would like to install the remaining Codewind component.
+5. Click **Install** to complete the Codewind installation. The file is <size of download>. For more information, see <more info link> (same link as in pop up). When installation completes, Codewind automatically connects to your local server and notifies you that it is on and not running.
 
-If you want to host or build the extension yourself, see [Contributing](#contributing).
+## Using Codewind for VS Code
+After the installation completes, you can use the following features in the Codewind workspace in the Explorer menu.
```

Comment:
I think this is kind of misleading; this is a very small subset of our available features. I think we should just have the Features list below.

Reply:
My concern here was that we should explain the buttons around the Codewind workspace as it appears in the Explorer menu...but now that you mention it, the hover help would probably provide sufficient explanation for these buttons. :)